library/core/src/ffi/c_str.rs

```diff
@@ -463,7 +463,7 @@ impl CStr {
     /// behavior when `ptr` is used inside the `unsafe` block:
     ///
     /// ```no_run
-    /// # #![allow(unused_must_use)] #![allow(temporary_cstring_as_ptr)]
+    /// # #![allow(unused_must_use)] #![allow(dangling_pointers_from_temporaries)]
```

Comment:
Due to bootstrapping we need to make the standard library build-able and test-able with the previous beta compiler. That can be achieved with the `bootstrap` config.
```suggestion
    /// # #![allow(unused_must_use)]
    /// # #![cfg_attr(bootstrap, allow(temporary_cstring_as_ptr))]
    /// # #![cfg_attr(not(bootstrap), allow(dangling_pointers_from_temporaries))]
```

Reply:
True, I forgot to mark it here. And forgot that cfg(bootstrap) exists in 25ee70b574cff3ea23e192b47a2bba9f144738e2. Thanks for the reminder.